dspace-spring-rest/src/main/java/org/dspace/app/rest/utils/Utils.java

```diff
@@ -219,4 +236,60 @@ public static String getFileName(MultipartFile multipartFile)
             return multipartFile.getName();
         }
     }
+
+    private List<DSpaceObject> constructDSpaceObjectList(Context context,
+                                                         HttpServletRequest request) throws IOException {
+        List<String> list = readFromRequest(request);
+
+        List<DSpaceObject> dSpaceObjects = new LinkedList<>();
+        for (String string : list) {
+            String uuid = string.substring(string.lastIndexOf('/') + 1);
```

Comment:
What would happen here if someone created a uri-list object that ended in a slash?

For example, `http://localhost:8080/api/core/collections/123-456-789/`.  Is there any better way to parse out the UUID?  Or just remove any trailing slashes?

Reply:
Perhaps:
String[] path = new URI(string).getPath().split("/");
String uuid = path[path.length - 1];

This can throw an exception if a URI is ill-formed, which is probably something we want to know.

Reply:
The trailing slashes are being handled now, but I added a more elaborate note on this implementation at 
https://github.com/DSpace/DSpace/pull/2332#discussion_r257778217